INFO.md

```diff
@@ -31,17 +31,25 @@ the repository's default branch on your behalf. Note that this requests the pull
 request creation in draft mode, essentially marking it as not yet ready to be
 reviewed.
 
+In the example above, we did not put the `/pr` command on the first line of the
+commit message. This means that it will not show up in the "short" format of the
+commit log. Users may chooto remove these command lines during squash, or
```

Comment:
did you mean to write _choose to_ instead of _chooto_?

Reply:
yes, choose is better than chooto